README.rst

```diff
@@ -74,6 +74,12 @@ If something has gone wrong, you may be prompted to manually install the fortran
 
   make -C sami2py/fortran compile
 
+or, on windows,
+
+.. code-block:: console
+
+  make -C sami2py\fortran compile
+
 Note that you will need a fortran compiler (gfortran is the default setup) and make installed on your system.
```

Comment:
Adding a section (here or elsewhere and linking here) on how to change the default compiler would be helpful.